array/two_sum_test.go

```diff
@@ -0,0 +1,18 @@
+package array
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestTwoSumFunc(t *testing.T) {
+	t.Log("Test Two Sum algorithm")
+	nums := []int{2, 7, 11, 15}
+	target := 9
+	fmt.Print("INPUT: ")
+	fmt.Print(nums)
+	fmt.Printf(", Target: %v", target)
+	fmt.Println()
```

Comment:
`fmt.Printf(", Target: %v\n", target)` is better than using `fmt.Println()` below `fmt.Printf`